Examples/Rendering/LabelmapMIP/index.js

```diff
@@ -0,0 +1,260 @@
+// Labelmap creation functions
+// ----------------------------------------------------------------------------
+
+function createRegularLabelmap(imageData, dims, center, radius1, radius2) {
+  const values = new Uint8Array(imageData.getNumberOfPoints());
+  const labelMapData = vtkImageData.newInstance(
+    imageData.get('spacing', 'origin', 'direction')
+  );
+
+  const dataArray = vtkDataArray.newInstance({
+    numberOfComponents: 1, // labelmap with single component
+    values,
+  });
+  labelMapData.getPointData().setScalars(dataArray);
+
+  labelMapData.setDimensions(...imageData.getDimensions());
+  labelMapData.setSpacing(...imageData.getSpacing());
+  labelMapData.setOrigin(...imageData.getOrigin());
+  labelMapData.setDirection(...imageData.getDirection());
+
+  const labelmapArray = labelMapData.getPointData().getScalars().getData();
+  const numberOfComponents = 1;
+
+  createSphericalLabel(
+    labelmapArray,
+    dims,
+    center,
+    radius1,
+    1,
+    numberOfComponents
+  );
+  createSphericalLabel(
+    labelmapArray,
+    dims,
+    [center[0] + 2 * radius1, center[1] + 2 * radius1, center[2] + 2 * radius1],
+    radius2,
+    2,
+    numberOfComponents
+  );
+
+  const labelmapActor = vtkVolume.newInstance();
+  const labelmapMapper = vtkVolumeMapper.newInstance();
+
+  // Labelmap pipeline
+  labelmapMapper.setInputData(labelMapData);
+  labelmapActor.setMapper(labelmapMapper);
+
+  const labelMapColorFunction = vtkColorTransferFunction.newInstance();
+  labelMapColorFunction.addRGBPoint(0, 0, 0, 0);
+  labelMapColorFunction.addRGBPoint(1, 0, 0, 1); // Blue for label 1
+  labelMapColorFunction.addRGBPoint(2, 1, 0, 0); // Red for label 2
+
+  const labelMapOpacityFunction = vtkPiecewiseFunction.newInstance();
+  labelMapOpacityFunction.addPoint(0, 0);
+  labelMapOpacityFunction.addPoint(1, 0.5);
+  labelMapOpacityFunction.addPoint(2, 0.5);
+
+  labelmapActor.getProperty().setRGBTransferFunction(0, labelMapColorFunction);
+  labelmapActor.getProperty().setScalarOpacity(0, labelMapOpacityFunction);
+
+  // set interpolation to nearest to avoid blending
+  labelmapActor.getProperty().setInterpolationTypeToNearest();
+  renderer.addVolume(labelmapActor);
+}
+
+function createAdvancedMIPLabelmap(imageData, dims, center, radius1, radius2) {
+  const array = imageData.getPointData().getArray(0);
+  const baseData = array.getData();
+
+  const numberOfComponents = 2;
+  const cubeData = new Float32Array(numberOfComponents * baseData.length);
+
+  // Copy original data to first component
+  for (let i = 0; i < baseData.length; i++) {
+    cubeData[i * numberOfComponents] = baseData[i];
+  }
+
+  createSphericalLabel(cubeData, dims, center, radius1, 1, numberOfComponents);
+  createSphericalLabel(
+    cubeData,
+    dims,
+    [center[0] + 2 * radius1, center[1] + 2 * radius1, center[2] + 2 * radius1],
+    radius2,
+    2,
+    numberOfComponents
+  );
+
+  actor.getProperty().setColorMixPreset(1);
```

Comment:
Can you replace `1` with `ColorMixPreset.ADDITIVE`?

Reply:
Done